### ISSUE TITLE
href relative paths are not proxied

### ISSUE THREAD

**Issue by Nottt** [Siujoeng-Lau/Workers-Proxy#14]
**Describe the bug**
If you proxy a subdomain with assets with relative links... example login/signup they will be loaded using the root domain, which is very often what we don't want. 

**To Reproduce**
Proxy a assets which contains relative paths for link href such a discourse forum.

**Screenshots**

**Additional context**
It would work OK if the worker rewrote all the relative paths of assets fetched with the domain we are fetching for the user

**Comment by Siujoeng-Lau** [comment]
Could you please attach your configuration of index.js?

**Comment by ParveenBhadooOfficial** [comment]
just copied and pasted the index.js file.

check https://proxy.bhadoo.workers.dev/

same problem.

**Comment by ParveenBhadooOfficial** [comment]
@Siujoeng-Lau is it possible to add an address bar and whatever url in the box we enter it should be the value of upstream const.

here is an example. but its in php
https://proxy.parveenbhadoo.com/proxy/

**Comment by Nottt** [comment]
Even that website fails to fully proxy the forum software I wanted...

https://proxy.parveenbhadoo.com/proxy/index.php?q=https%3A%2F%2Fmeta.discourse.org

**Comment by ParveenBhadooOfficial** [comment]
@Nottt Yes, that's why we need this one here. it still works for many websites completely.

**Comment by Nottt** [comment]
I'm not sure what you mean

**Comment by ParveenBhadooOfficial** [comment]
@Nottt I mean proxy.parveenbhadoo.com works for many websites like zooqle.com but it's buggy. so we need Workers-Proxy or alternative.

**Comment by Nottt** [comment]
This project also fails to properly proxy meta.discourse.org
